### PR TITLE
add ability to create an index after the import

### PR DESCRIPTION
**Pull request by maguec**
add -i / --index to create an index after the import

test included to ensure the explain properly shows the index being used

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ python3 redisgraph_bulk_loader/bulk_insert.py GRAPHNAME [OPTIONS]
 |  -t   | --max-token-count INT      |            (Debug argument) Max number of tokens sent in each Redis query (default 1024)             |
 |  -b   | --max-buffer-size INT      |               (Debug argument) Max batch size (MBs) of each Redis query (default 4096)               |
 |  -c   | --max-token-size INT       |              (Debug argument) Max size (MBs) of each token sent to Redis (default 500)               |
+|  -i   | --index Label:Property     |              After bulk import create an Index on Label and Propery defined (optional)               |
```
Comment (jeffreylovitz) [inline README.md:57]: Typo - `Property defined`
I'd also suggest rewording to something like:
`After bulk import, create an Index on provided Label:Property pair`

File: redisgraph_bulk_loader/bulk_insert.py [hunk 2]
```diff
@@ -62,7 +62,8 @@ def process_entities(entities):
 @click.option('--max-token-count', '-c', default=1024, help='max number of processed CSVs to send per query (default 1024)')
 @click.option('--max-buffer-size', '-b', default=2048, help='max buffer size in megabytes (default 2048)')
 @click.option('--max-token-size', '-t', default=500, help='max size of each token in megabytes (default 500, max 512)')
-def bulk_insert(graph, host, port, password, nodes, nodes_with_label, relations, relations_with_type, separator, enforce_schema, skip_invalid_nodes, skip_invalid_edges, quote, max_token_count, max_buffer_size, max_token_size):
+@click.option('--index', '-i', multiple=True, help='Label:Propery on which to create an index')
```
Comment (jeffreylovitz) [inline redisgraph_bulk_loader/bulk_insert.py:65]: Typo - `Property`

File: test/test_bulk_loader.py [hunk 3]
```diff
@@ -660,6 +660,31 @@ def test16_error_on_schema_failure(self):
             self.assertEqual(sys.exc_info()[0].__name__, 'SchemaError')
             self.assertIn("Could not parse 'strval' as an array", e.args)
 
+    def test17_ensure_index_is_created(self):
```
Comment (jeffreylovitz) [inline test/test_bulk_loader.py:663]: Nice test!

**Review by jeffreylovitz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (maguec) [reply README.md]: fixxored

**Review by jeffreylovitz — APPROVED** [review]
Looks great, thanks!
